src/sentry/integrations/msteams/card_builder/block.py

```diff
@@ -42,7 +32,7 @@ class ImageSize(str, Enum):
     MEDIUM = "Medium"
     LARGE = "Large"
     AUTO = "Auto"
-    STRECH = "Strech"
+    STRETCH = "Stretch"
```

Comment:
this was typo'd -- noticed while adapting the `incident_attachment` card to fit the type system